content/log/processing.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Processing
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview
+
+To access the processing panel please use the upper left menu:
+{{< img src="log/processing/processing_panel.png" alt="Pipelines pannel" >}}
+
+Your processing view should look like this:
+{{< img src="log/processing/processing_view.png" alt="Pipelines view" >}}
+
+## Processing Pipelines 
+### Pipelines Goal 
+
+**A processing pipeline applies over a filtered subset of incoming logs a list of sequential processors.**
+
+With pipelines you have the opportunity to parse & enrich your logs through a sequential chains of [processors](#processors) in order to extract meaningful information or to extract attributes from semi-structured text to re-use them as [facet](/log/explore/#facets).
+
+Each log that comes through the pipelines is tested against every pipeline filter. If it matches one then all the [processors](#processors) are applied sequentially before moving to the next pipeline.
+
+So for instance a processing pipeline can transform this log:
+{{< highlight json >}}
+{
+"message":"john clicked on a link : www.datadoghq.com for app dd_app",
+"hostname":"host_1",
+"level":"info"
+}
+{{</ highlight >}}
+
+into this log: 
+
+{{< highlight json >}}
+{
+"user":"john",
+"url":"www.datadoghq.com",
+"appname":"dd_app",
+"message":"clicked on a link",
+"hostname":"host_1",
+"severity":"info"
+}
+{{< /highlight >}}
+
+With one single pipeline as follow:
+
+{{< img src="log/processing/pipeline_processors_sketch.png" alt="Pipelines processor sketch" style="width:90%;">}}
+
+Pipelines are also able to take all your logs whatever they source and transform them to handle a unifed and structured format on Datadog.
+
+For instance, in the following example we have at the begining 3 differents raw logs:
+
+{{< img src="log/processing/multi_source_sketch.png" alt="Multi source sketch" >}}
+
+and at the end we have the same information, but with more coherence that allow a better usability.
+
+### Pipeline filters
+
+Attach a filter to your pipeline in order to apply it only to a subset of your logs. 
+
+The filter syntax is the same as the [search bar](/log/explore/#search-bar).
+
+**Be aware that the pipeline filtering is applied before any pipeline processing, hence you can not filter on an attribute that does not exists** 
```

Comment:
* s/can not/cannot
* s/exists/exist

Also, I'm not sure I understand how the second half of this sentence—"hence you can not filter on an attribute that does not exists"—follows from the first.

Reply:
A pipeline allows you to extract value (text) from a log and assign it to a given attribute.
but since it's doing this within a pipeline you cannot use this attribute as a filter for the pipeline.